bucket/ipopt.json

```diff
@@ -0,0 +1,29 @@
+{
+    "version": "3.13.2",
+    "description": "Software package for large-scale nonlinear optimization",
+    "homepage": "https://github.com/coin-or/Ipopt",
```

Comment:
...

```suggestion
    "homepage": "https://coin-or.github.io/Ipopt/",
```

![A](https://i.imgur.com/24h9xTV.png)

Reply:
That's what I put at first but it is written in the readme "The Ipopt project webpage is https://github.com/coin-or/Ipopt."

Reply:
No.

Reply:
Fine.